crates/rpc-ws-server/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "gw-rpc-ws-server"
+version = "0.1.0"
+authors = ["Nervos network"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+ckb-types = "0.100.0"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+serde_plain = "0.3.0"
+jsonrpc-core = "18.0"
+jsonrpc-derive = "18.0"
+jsonrpc-http-server = "18.0"
+jsonrpc-tcp-server = "18.0"
```

Comment:
jsonrpc-http-server = "18.0"
jsonrpc-tcp-server = "18.0"
Looks like we don't need both crates ?